src/houston/public/styles/houston.css

```diff
@@ -8,37 +8,28 @@
 ******/
 
 nav {
-    background-color: #3a416f;
+    background-color: #403757;
```

Comment:
Yes, the `houston.css` file applies to all the pages, which most of them use the colored header. The `index.css` should overwrite that style to match the current website developer page.